source/includes/_budget.md

```diff
@@ -2,31 +2,43 @@
 
 ## Budget Object
 
-Attribute Name      | Type   | Description
-------------------- | ----   | -----------
-category_name       | string | Name of the category
-category_id         | number | Unique identifier for category
-category_group_name | string | Name of the category group, if applicable
-group_id            | number | Unique identifier for category group
-is_group            | boolean | If true, this category is a group
-is_income           | boolean | If true, this category is an income category (category properties are set in the app via the Categories page)
-exclude_from_budget | boolean | If true, this category is excluded from budget (category properties are set in the app via the Categories page)
-exclude_from_totals | boolean | If true, this category is excluded from totals (category properties are set in the app via the Categories page)
-data                | Array of data objects | For each month with budget or category spending data, there is a data object with the key set to the month in format YYYY-MM-DD. For properties, see Data object below.
-config              | object | Object representing the category's budget suggestion configuration
+Attribute Name      | Type   | Nullable | Description
+------------------- | ----   | -------- | -----------
+category_name       | string | false | Name of the category
+category_id         | number | true | Unique identifier for category
```

Comment:
category_id should not be nullable!

Reply:
So this one surprised me, but it looks like it can be null with a category_name of `Uncategorized`. Example if I call get budget summary for my account I have some entries like this:


```
{:category_name=>"Uncategorized",
  :category_id=>nil,
  :is_group=>false,
  :group_id=>nil,
  :is_income=>false,
  :exclude_from_budget=>false,
  :exclude_from_totals=>false,
  :archived=>false,
  :data=>{:"2023-02-01"=>{:num_transactions=>13, :spending_to_base=>123.4567889999999}, :"2023-01-01"=>{:num_transactions=>24, :spending_to_base=>1234.56}},
  :config=>nil,
  :order=>0}
```

Reply:
Ah, good catch! Can we make a note of that in the description perhaps?